docs/products/redis/concepts/ssl-connectivity.rst

```diff
@@ -0,0 +1,85 @@
+
+.. Tip::
+    You can find the ``Service URI`` on `Aiven console <https://console.aiven.io/>`_.
+
+Since **Redis 6**, the redis-cli tool itself supports SSL connections; therefore, you can connect directly to your service using::
+
+    redis-cli -u rediss://username:password@host:port
+
+Alternatively, you can use the third-party `redli tool <https://github.com/IBM-Cloud/redli>`_::
+
+    redli -u rediss://username:password@host:port
+
+
+Not every Redis client supports SSL-encrypted connections.
+In such cases, you would have to turn off SSL to use these clients, which is allowed but not recommended. In those cases, you have some options:
+
+
+Set up stunnel process
+~~~~~~~~~~~~~~~~~~~~~~
+
+One workaround for this, it is to set up a `stunnel process <https://www.stunnel.org/index.html>`_ on the client side to handle encryption for the clients that do not support SSL connections. 
+
+You can use the following stunnel configuration, for example ``stunnel.conf``, to set up a stunnel process.
+::
+
+    client = yes
+    foreground = yes
+    debug = info
+    delay = yes
+
+    [redis]
+    accept = 127.0.0.1:6380
+    connect = myredis.testproject.aivencloud.com:28173
+    TIMEOUTclose = 0
+    ; For old services only. New ones use Let's Encrypt and there's no
+    ; CA cert available from Aiven console. Most environments trust
+    ; Let's Encrypt by default without any explicit CAfile config.
+    ; CAfile = /path/to/optional/project/cacert/that/you/can/download/from/aiven/console
+
+To understand the global options of the stunnel configuration, please check `Stunnel Global Options <https://www.stunnel.org/static/stunnel.html#GLOBAL-OPTIONS>`_.
+
+For ``service-level option``, the following parameters are configured:  
+
+``accept`` => *[host:]port*
+  **Accept connections on specified address**
+
+
+
+``connect`` => *[host:]port*
+  **Connect to a remote address** 
+
+
+
+``TIMEOUTclose`` => *seconds*
+  **Time to wait for close_notify**
+
+.. note:: It is important to make changes accordingly to your service. On the *Overview* page you can find your **Overview** > **Host** and **Overview** > **Port** to configure the ``connect`` parameter.
+
+Note that when SSL is in use we have a separate service terminating the SSL connections before they are forwarded to Redis. This process has a connection timeout of its own independent of Redis' connection timeout. If you allow very long Redis timeouts, this frontend service may end up closing the connection before the Redis timeout has expired. By the time of writing this, timeout is set to 12 hours.
+
+Allow plain-text connections
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Another alternative is to actually allow plain-text connections. To allow plain-text connections, you can change this setting with **Overview** > **Advanced configuration**, or using the :doc:`Aiven Command Line interface<../../../tools/cli>`.
```

Comment:
```suggestion
The alternative is to disable database SSL allowing plain-text connections. To allow plain-text connections, you can change this setting with **Overview** > **Advanced configuration**, or using the :doc:`Aiven Command Line interface<../../../tools/cli>`.
```

Reply:
I guess this is one of the alternatives not the only one, so imho `The alternative` may be not the proper term.

Reply:
You could use "An alternative" here if you prefer. It's up to you whether you accept the suggested wording change (and you can make more changes after you do, if you want)

Reply:
oki, I will use `An alternative`